lib/moddle.js

```diff
@@ -141,6 +141,12 @@ Moddle.prototype.createAny = function(name, nsUri, properties) {
     $type: name,
     $instanceOf: function(type) {
       return type === this.$type;
+    },
+    get: function(name) {
+      return this[name];
+    },
+    set: function(name, value) {
+      this[name] = value;
```

Comment:
This leads to prototype pollution:

```
~/workspace/bpmn-io/moddle replicate-api-on-createAny 7s
❯ node
Welcome to Node.js v18.12.1.
Type ".help" for more information.
> const Moddle = require('.')
undefined
> Moddle
{
  Moddle: [Function: Moddle],
  coerceType: [Function: coerceType],
  isBuiltInType: [Function: isBuiltIn],
  isSimpleType: [Function: isSimple],
  parseNameNS: [Function: parseName]
}
> const model = new Moddle.Moddle([])
undefined
> const a = model.createAny('test', 'http://example.com')
undefined
> a.set('__proto__', { hacked() { console.log('hacked') } })
undefined
> a.hacked()
hacked
undefined
```

Cf. `min-dash#set` for mitigation: https://github.com/bpmn-io/min-dash/blob/master/lib/object.js#L40
We could also reuse min-dash method if we make sure `name` is string and not array.

Reply:
Let's use `min-dash#set` here.

Reply:
Good catch. Addressed with [118b5fd](https://github.com/bpmn-io/moddle/pull/54/commits/118b5fdc4d8859c689120b529022712627492b93) by using min-dash